src/classiceditorui.js

```diff
@@ -29,20 +29,24 @@ export default class ClassicEditorUI extends BoxedEditorUI {
 		super( editor );
 
 		/**
-		 * Toolbar controller.
+		 * Editable UI controller.
 		 *
 		 * @readonly
-		 * @member {ui.toolbar.Toolbar} editor-classic.ClassicEditorUI#toolbar
+		 * @member {ui.editableUI.EditableUI} editor-classic.ClassicEditorUI#editable
 		 */
-		this.toolbar = this._createToolbar();
+		this.editable = this._createEditableUI();
+	}
 
+	init() {
 		/**
-		 * Editable UI controller.
+		 * Toolbar controller.
 		 *
 		 * @readonly
-		 * @member {ui.editableUI.EditableUI} editor-classic.ClassicEditorUI#editable
+		 * @member {ui.toolbar.Toolbar} editor-classic.ClassicEditorUI#toolbar
 		 */
-		this.editable = this._createEditableUI();
+		this.toolbar = this._createToolbar( this.editor.ui.view.element );
```

Comment:
`StickyToolbarView` needs `limiterElement` to stick properly. In case of `ClassicEditorUi` it's `this.editor.ui.view.element`, which is unavailable in `ClassicEditorUi#constructor()` yet. 

So the toolbar must be created either in `ClassicEditorUi#init()` or `limiterElement` must be moved to `StickyToolbarViewModel#limiterElement` and set in `ClassicEditorUi#init()`. 

Either way, there's some job to be done in `ClassicEditorUi#init()` and I'm wondering if `limiterElement` should be part of component's state. This was, in fact, what the first implementation looked like but I backed out of this approach. IMO `StickyToolbarView` is way too static, i.e. it depends on its `_placeholderElement` to work properly, which is injected once where the toolbar is created and never moves in DOM, so making `limiterElement` part of the model makes little sense, because the toolbar would not be able to stick somewhere else anyway, it's bound to the spot it's been initialised, so why bother.